### PR TITLE
Feat:"Added Playlist Page"

### LINKED ISSUE #15: Feat : "Playlist Page"

### PR DESCRIPTION
**Pull request by sadathshariff**
Implemented 
User can add Videos to Playlist, delete the Playlist and Delete Video From the Playlist
Single Playlist Page

### CODE REVIEW

**Review by therajatg — APPROVED** [review]
code looks good

File: src/components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  top: 0;
+  left: 0;
+  padding: 0.5rem;
+  margin-top: 0.5rem;
+  border-radius: 10px;
+  position: fixed;
+  z-index: 3;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (nishasen) [inline src/components/Modal/Modal.css:12]: use util class here

File: src/components/Modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.modal-container {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  padding: 0.5rem;
+  margin-top: 0.5rem;
+  border-radius: 10px;
+  position: fixed;
```
Comment (nishasen) [inline src/components/Modal/Modal.css:9]: you have given position: fixed two times

File: src/components/Modal/Modal.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  width: 25rem;
+  padding: 1rem;
+  border-radius: 10px;
+  background-color: var(--text-color);
+}
+
+.modal-div {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (nishasen) [inline src/components/Modal/Modal.css:27]: use util class

File: src/components/VideoCard/VideoCard.jsx [hunk 4]
```diff
@@ -14,20 +15,34 @@ import {
...
   return (
     <>
-      <div className="videoCard">
+      <motion.div
+        layout
+        animate={{ opacity: 1 }}
+        initial={{ opacity: 0 }}
+        exit={{ opacity: 0 }}
+        className="videoCard"
+      >
```
Comment (nishasen) [inline src/components/VideoCard/VideoCard.jsx:45]: Learnt something new here

**Review by krishanu2911 — APPROVED** [review]
well written code 
Nicely Divided into components
Good to go👍🏻

**Review by nishasen — APPROVED** [review]
Looks good , suggested some small changes

File: src/components/Modal/Modal.css (hunk 2 above)
Comment (sadathshariff) [reply src/components/Modal/Modal.css:9]: Thanks Nisha for pointing it out

**Pull request merged**